runtime/vm/intfunc.c

```diff
@@ -376,6 +376,7 @@ J9InternalVMFunctions J9InternalFunctions = {
 #endif /* J9VM_OPT_JITSERVER */
 	createJoinableThreadWithCategory,
 	valueTypeCapableAcmp,
+	isNameOrSignatureQtype,
 	isClassRefQtype,
```

Comment:
do you still have any use for `isClassRefQtype`? it was added for the JIT. If it is not longer needed you can remove it from the table

Reply:
`isClassRefQtype` is used in JIT such as in the following places: 

https://github.com/eclipse/openj9/blob/5d43290a25c5b29c3c2b0571b862820877013b70/runtime/compiler/ilgen/Walker.cpp#L2378
https://github.com/eclipse/openj9/blob/5d43290a25c5b29c3c2b0571b862820877013b70/runtime/compiler/ilgen/IlGenerator.cpp#L570